validator/accounts/cmd_accounts.go

```diff
@@ -171,6 +173,11 @@ this command outputs a deposit data string which is required to become a validat
 			Action: func(cliCtx *cli.Context) error {
 				featureconfig.ConfigureValidator(cliCtx)
 				if err := ExitAccountsCli(cliCtx, os.Stdin); err != nil {
+					msg := err.Error()
```

Comment:
@rkapka let's use `errors.Is` instead of doing string comparisons

Reply:
I don't think `errors.Is` can be used here as I am not comparing errors. The exported messages are plain strings.